File: 151001/Danko/RESTApplication/src/main/java/com/example/restapplication/dto/UserResponseTo.java
package com.example.restapplication.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserResponseTo {
    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 151003/Barylko/Kafka/Module.Publisher/Dto/Response/UserResponseTo.cs
using System.Text.Json.Serialization;
namespace Publisher.Dto.Response;

public record UserResponseTo(
    [property: JsonPropertyName("id")] 
    long Id,
    [property: JsonPropertyName("login")] 
    string Login,
    [property: JsonPropertyName("password")]
    string Password,
    [property: JsonPropertyName("firstname")]
    string FirstName,
    [property: JsonPropertyName("lastname")]
    string LastName
);